src/main/java/com/puppycrawl/tools/checkstyle/checks/AvoidEscapedUnicodeCharactersCheck.java

```diff
@@ -96,42 +92,63 @@
  * &lt;/module&gt;
  * </pre>
  * <p>
- * Example of using escapes with trail comment:
+ * Example of using escapes for non-printable, control characters:
  * </p>
  * <pre>
- * String unitAbbrev = "&#92;u03bcs"; // Greek letter mu, "s"
- * String textBlockUnitAbbrev = """
- *          &#92;u03bcs"""; // Greek letter mu, "s"
+ * String unitAbbrev = "μs";      // OK, a normal String
+ * String unitAbbrev = "&#92;u03bcs"; // violation, "&#92;u03bcs" is a printable character.
+ * return '&#92;ufeff' + content;     // OK, non-printable control character.
  * </pre>
- * <p>An example of how to configure the check to allow using escapes
+ * <p>
+ * An example of how to configure the check to allow using escapes
  * if trail comment is present:
  * </p>
  * <pre>
  * &lt;module name="AvoidEscapedUnicodeCharacters"&gt;
  *   &lt;property name="allowByTailComment" value="true"/&gt;
  * &lt;/module&gt;
  * </pre>
- * <p>Example of using escapes if literal contains only them:
+ * <p>Example of using escapes if trail comment is present:
  * </p>
  * <pre>
- * String unitAbbrev = "&#92;u03bc&#92;u03bc&#92;u03bc";
+ * String unitAbbrev = "μs";      // OK, a normal String
+ * String unitAbbrev = "&#92;u03bcs"; // OK, Greek letter mu, "s"
+ * return '&#92;ufeff' + content;
+ *                                // violation, comment is not used within same line.
```

Comment:
`violation, on previous line, comment is..`

Reply:
This example is for allow by tail comment, if I use a comment on the same line then it wouldn't be a violation. That's why I have not used the comment on the same line.
```
// ---------^------------- violation, comment is not used within same line.
```
I will do something like this to make it more clear

Reply:
I know, that is why I requested extra words like"on previous line"